io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,43 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
```

Comment:
what is sound for?

Reply:
When a break is done, a notification sound is played since the user is probably not looking at their screen during the break.